content/graph/lca.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * Author: Thomas Meeks
+ * Date: 2024-10-08
+ * Source: own
+ * Description: Given a tree creates structure allowing for 
+ * O(log(n)) lca queries between arbitary nodes (root = 0)
```

Comment:
While the current description does mention "LCA", it doesn't at all mention what LCA means/stands for. "Lowest common ancestor" should probably be explicitly stated somewhere.